src/components/Icons/index.tsx

```diff
@@ -28,6 +28,7 @@ export const Play = <CaretRightOutlined />;
 export const Pause = <PauseOutlined />;
 export const UploadFile = <ImportOutlined />;
 export const DownArrow = <DownOutlined />;
+export const UpArrow = <DownOutlined style={{ transform: "rotate(180deg" }} />;
```

Comment:
Cheeky, but I'm not complaining.

Reply:
Is that paren supposed to be closed? I see `rotate(180deg`, not `rotate(180deg)`

Reply:
Weirdly seems to work without closing it, but awful to look at, I'll close it.